jenkinsfiles/Jenkinsfile.grobid

```diff
@@ -28,5 +28,11 @@ elifePipeline {
             unstable_image.tag(tag).push()
             unstable_image.push()
         }
+
+        stage 'Trigger grobid tag update', {
+            build job:  '../dependencies/dependencies-sciencebeam-trainer-update-grobid',
+                        wait: false,
+                        parameters: [string(name: 'grobid_tag', value: commit)]
```

Comment:
https://alfred.elifesciences.org/job/dependencies/job/dependencies-sciencebeam-trainer-update-grobid/4/replay/ doesn't have a default for that, could be added there because in its own Jenkins definition it is defined without parameters. So you get a `Build now` button rather than `Build with parmeters` that may break when pressed if that pipeline code doesn't deal with a null value.

Reply:
i guess the expected behavior is that the pipeline should break when no parameter is provided - @de-code (correct me if i am wrong , please)

Reply:
Good for it to break if there's no default. If you add the parameter definition in `This project is parameterised` in https://alfred.elifesciences.org/job/dependencies/job/dependencies-sciencebeam-trainer-update-grobid/configure it will make this requirement visible in the Jenkins UI.

Reply:
done! thanks